### PR TITLE
yutto: init at 2.0.0b13

### PR DESCRIPTION
**Pull request by linsui**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -30940,6 +30942,8 @@ with pkgs;
 
   ytmdl = callPackage ../tools/misc/ytmdl { };
 
+  yutto = callPackage ../tools/misc/yutto { };
```
Comment (kira64xyz) [inline pkgs/top-level/all-packages.nix:30945]: Couldn't we use `toPythonApplication` here as well?

File: pkgs/tools/misc/yutto/default.nix [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  propagatedBuildInputs = [
+    aiohttp
+    aiofiles
+    biliass
+    dicttoxml
+    colorama
+  ];
+
+
```
Comment (kira64xyz) [inline pkgs/tools/misc/yutto/default.nix:33]:
```suggestion
  pythonImportsCheck = [ module_name ];
```
See [the documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck-using-pythonimportscheck) for more information

**Review by kira64xyz — CHANGES_REQUESTED** [review]
Just a few little things :blush:

File: pkgs/tools/misc/yutto/default.nix (hunk 2 above)
Comment (linsui) [reply pkgs/tools/misc/yutto/default.nix:33]: Thanks!

File: pkgs/top-level/all-packages.nix (hunk 1 above)
Comment (linsui) [reply pkgs/top-level/all-packages.nix:30945]: I thought yutto is an application instead of a library. Should I put it into python-packages.nix?

Comment (kira64xyz) [reply pkgs/top-level/all-packages.nix:30945]: Exactly

File: pkgs/development/python-modules/biliass/default.nix [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, protobuf
+}:
+
+buildPythonPackage rec {
```
Comment (kira64xyz) [inline pkgs/development/python-modules/biliass/default.nix:8]: Should add `pythonImportsCheck` here too

**Review by kira64xyz — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/biliass/default.nix (hunk 3 above)
Comment (linsui) [reply pkgs/development/python-modules/biliass/default.nix:8]: Done.

File: pkgs/development/python-modules/biliass/default.nix [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-kktK+6rLwYhkG7LiTBlgBbiIN8apweg4l8pJSTjKQU4=";
+  };
+
+  propagatedBuildInputs = [ protobuf ];
+
+  pythonImportsCheck = [ pname ];
```
Comment (kira64xyz) [inline pkgs/development/python-modules/biliass/default.nix:20]: You should type the name out instead of using `pname`, for readability's sake

**Review by kira64xyz — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/biliass/default.nix (hunk 4 above)
Comment (linsui) [reply pkgs/development/python-modules/biliass/default.nix]: Done.

**Review by kira64xyz — APPROVED** [review]
Looks good to me :relieved: 

Result of `nixpkgs-review pr 173134` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>biliass (python39Packages.biliass)</li>
    <li>python310Packages.biliass</li>
    <li>python310Packages.yutto</li>
    <li>yutto (python39Packages.yutto)</li>
  </ul>
</details>

**Review by IvarWithoutBones — APPROVED** [review]

**Pull request merged**